compositing-2/Overview.bs

```diff
@@ -709,16 +709,15 @@ If the 'knock-out' property is disabled, each element within the group is only c
 -->
 <h3 id="pagebackdrop">The Page Group</h3>
 
-The <a href="#isolatedgroups">isolated group</a for the root element is the
+The <a href="#isolatedgroups">isolated group</a> for the root element is the
 page group. All other elements and groups are composited into this group.
 The background of the root element (if specified) is painted into the page
 group, and any filter, clip-path, mask and and opacity is then applied, before
 compositing into the <a href="#rootgroup">root group</a>.
 
 <h3 id="rootgroup">The Root Group</h3>
 The root group is the final compositing output, and is is above the page
```

Comment:
the root group is not "output". It's just a group with a UA defined color that also contains the page group.

Typically this process is compositing with a solid opaque color is called "matting". Describing the process using groups is ok though, although slightly more complicated.

Reply:
I rephrased to not say "output".

Reply:
This is the link I was suggesting: https://www.w3.org/TR/css-color-4/#sample

Reply:
I think it's still confusing. Maybe:
`The root group encompasses the entire canvas and contains (or is below) the page group`

Reply:
Updated, to this phrasing, thanks.